src/mtx/CommandBinaryBuffer.ts

```diff
@@ -1,4 +1,314 @@
+    RELAY_OFF_LIM_TARIFF_3: boolean,
+    RELAY_OFF_PF_MIN: boolean
+}
+
+export interface IRelaySet4OperatorParameter {
+    RELAY_ON_TIMEOUT: boolean,
+    RELAY_ON_SALDO: boolean,
+    RELAY_OFF_SALDO: boolean,
+    RELAY_OFF_SALDO_SOFT: boolean,
+    RELAY_OFF_MAGNET: boolean,
+    RELAY_ON_MAGNET_TIMEOUT: boolean,
+    RELAY_ON_MAGNET_AUTO: boolean
+}
+
+export interface IRelaySet5OperatorParameter {
+    RELAY_OFF_UNEQUAL_CURRENT: boolean,
+    RELAY_ON_UNEQUAL_CURRENT: boolean,
+    RELAY_OFF_BIPOLAR_POWER: boolean,
+    RELAY_ON_BIPOLAR_POWER: boolean
+}
+
+export interface IDefine1OperatorParameter {
+    BLOCK_KEY_OPTOPORT: boolean,
+    MAGNET_SCREEN_CONST: boolean
+}
+
+export interface IOperatorParameters {
+    vpThreshold: number,
+    vThreshold: number,
+    ipThreshold: number,
+    pmaxThreshold0: number,
+    pmaxThreshold1: number,
+    pmaxThreshold2: number,
+    pmaxThreshold3: number,
+    speedOptoPort: number,
+    tint: number,
+    calcPeriodDate: number,
+    timeoutDisplay: number,
+    timeoutScreen: number,
+    displaySet: IDisplaySetOperatorParameter,
+    relaySet4: IRelaySet4OperatorParameter,
+    relaySet3: IRelaySet3OperatorParameter,
+    relaySet2: IRelaySet2OperatorParameter,
+    relaySet1: IRelaySet1OperatorParameter,
+    displayType: number,
+    ten: number,
+    timeoutRefresh: number,
+    deltaCorMin: number,
+    timeoutMagnetOff: number,
+    timeoutMagnetOn: number,
+    define1: IDefine1OperatorParameter,
+    timeoutRelayOn: number,
+    timeoutRelayKey: number,
+    timeoutRelayAuto: number,
+    timeoutBadVAVB: number,
+    freqMax: number,
+    freqMin: number,
+    phMin: number,
+    year: number,
+    month: number,
+    date: number,
+    energyDecimalPoint: number,
+    typeMeter: number,
+    timeoutIMax: number,
+    timeoutPMax: number,
+    timeoutCos: number,
+    pMaxDef: number,
+    displaySetExt: IDisplaySetExtOperatorParameter,
+    timeoutUneqCurrent: number,
+    timeoutBipolarPower: number,
+    relaySet5: IRelaySet5OperatorParameter,
+    timeCorrectPeriod: number,
+    timeCorrectPassHalfhour: boolean,
+}
+
+
+export const TARIFF_PLAN_SIZE = 11;
+export const OPERATOR_PARAMETERS_SIZE = 74;
+
+const displaySetMask = {
+    TEST_D: 0x0001,
+    VERSION_D: 0x0002,
+    WH_D: 0x0004,
+    WH_T1_D: 0x0008,
+    WH_T2_D: 0x0010,
+    WH_T3_D: 0x0020,
+    WH_T4_D: 0x0040,
+    POWER_D: 0x0080,
+    POWERB_D: 0x0100,
+    IRMS_D: 0x0200,
+    IRMSB_D: 0x0400,
+    VRMS_D: 0x0800,
+    TIME_D: 0x1000,
+    DATE_D: 0x2000,
+    WH_EXP_D: 0x4000,
+    WH_EXP_T1_D: 0x8000,
+    WH_EXP_T2_D: 0x00010000,
+    WH_EXP_T3_D: 0x00020000,
+    WH_EXP_T4_D: 0x00040000,
+    PF_A: 0x00080000,
+    PF_B: 0x00100000,
+    VBAT: 0x00200000,
+    POWER_THRESHOLD_T0: 0x00400000,
+    POWER_THRESHOLD_T1: 0x00800000,
+    POWER_THRESHOLD_T2: 0x01000000,
+    POWER_THRESHOLD_T3: 0x02000000,
+    SALDO_D: 0x20000000,
+    SCROLL_D: 0x80000000
+};
+
+const displaySetExtMask = {
```

Comment:
`displaySetExtMask` and `displaySetMask` have common part (displaySetMask without SCROLL_D) maybe create general/base mask `displaySetMaskBase`?

Reply:
done